src/ipc/ipc.c

```diff
@@ -401,15 +401,15 @@ int ipc_comp_connect(struct ipc *ipc,
 	/* check whether the components already exist */
```

Comment:
small typos in commit message: connet, exisct

Reply:
Updated